classes/template/PKPTemplateManager.inc.php

```diff
@@ -758,15 +758,7 @@ function registerJSLibraryData() {
 		}
 
 		// Allow plugins to load data within their own namespace
-		$pluginData = array();
-		HookRegistry::call('TemplateManager::registerJSLibraryData', array(&$pluginData));
```

Comment:
(I'm not aware of this being used anywhere else -- but we should be cautious about removing hooks in general)

Reply:
In this case, the hook was unusable because it was being fired before other plugins were able to hook in. It was being used by the TinyMCE plugin but to no effect. Figuring this out cost me a full morning. :)

Reply:
Yikes! I shouldn't have asked...